### PR TITLE
Add documentation for merchant segregation

### PR DESCRIPTION
**Pull request by ABEK-MP**

### CODE REVIEW

File: docs/endpoint_errors.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ This page contains information regarding all the non-successful status-codes and
 | StatusCode | ErrorCodes  | Description |
 |------------|-------------|-------------|
 | 400 | 1102`` `` <br> 1105`` `` <br> 1117`` `` | Invalid ``Amount`` <br> Invalid ``UserMinimumAge`` <br> Invalid ``MerchantPaymentLabel`` |
-| 403 | 1401 | Cannot ready payments prepared by a different integrator |
+| 403 | 1401<br>1406 | Cannot ready payments prepared by a different integrator<br>Cannot ready payment prepared on behalf of a different merchant |
```
Comment (ChristianDoormann) [inline docs/endpoint_errors.md:64]: payments in plural :)

File: docs/pos_management.md [hunk 2]
```diff
@@ -67,4 +67,4 @@ We recommend the client to store the following in a configuration file to be abl
...
 
-A store is identified by a ````storeId````, but it is also possible to refer to a store by the combination of ````merchantBrandId```` and ````merchantLocationId````. Two stores with the same ````merchantLocationId```` but different ````merchantBrandId````s are not related in any way. The ````merchantBrandId````, ````merchantLocationId```` and ````storeId```` are supplied by MobilePay when the Merchant/Store is onboarded. 
+A store is identified by a ````storeId````, but it is also possible to refer to a store by the combination of ````merchantBrandId```` and ````merchantLocationId````. Two stores with the same ````merchantLocationId```` but different ````merchantBrandId````s are not related in any way. The ````merchantBrandId````, ````merchantLocationId```` and ````storeId```` are supplied by MobilePay when the Merchant/Store is onboarded.
```
Comment (ChristianDoormann) [inline docs/pos_management.md:70]: what changed in this phrase? - I dont see any difference

Comment (ABEK-MP) [reply docs/pos_management.md:70]: Trailing newline issue or something?

File: docs/release_notes.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ The Point of Sale API V10 is now in production.
 
 ## Changelog
 
+### 2020-10-20
+- Added merchant segregation: integrators can now only perform actions on entities on behalf of the merchants that they were created on behalf of.
```
Comment (ChristianDoormann) [inline docs/release_notes.md:8]: Maybe hide the fact that we did not support this before. So maybe just write that we now have added error codes for merchant segregation and link to the input errors that you already do.

**Pull request merged**